modules/ROOT/pages/FAQ/index.adoc

```diff
@@ -647,6 +647,45 @@ will be published as a Change notice for a link:https://docs.ted.europa.eu/eform
 
 
 
+What is the correct procedure when creating a Contract Modification Notice with multiple changes, in particular, regarding Modification Previous Notice Section Identifier (BT-1501)? How should we reference previous Contract Modification Notices?::
+
+You can consult the https://docs.ted.europa.eu/eforms/latest/schema/contract-modification-notice.html[TED Developer Docs]. As with Change notices, the Contract Modification notice should contain the consolidated
+information for a given contract. A Contract Modification notice can only change one contract at a time 
+and should only contain the information relevant to the modified contract, i.e. other contracts and lots 
```

Comment:
Contracts and lots ern't the only ones to be removed, it is also the case for thenders, tenderers, organisations ...
"... modified contract(s) ... other Contracts, lots etc. ..."